src/components/ebay-menu-button/examples/18-typeahead/template.marko

```diff
@@ -0,0 +1,53 @@
+<style>
```

Comment:
You can removed this style section from this example as it's not used.

Reply:
done